spec/defines/windows_firewall/exception_spec.rb

```diff
@@ -387,16 +387,16 @@
       let(:title) { 'Windows Remote Management' }
       let :params do
         {
-          ensure: 'present', direction: 'fubar', action: 'allow', enabled: 'yes',
-          protocol: 'TCP', local_port: '5985',
+          ensure: 'present', direction: 'fubar', action: 'allow', enabled: true,
+          protocol: 'TCP', local_port: 5985,
           display_name: 'Windows Remote Management', description: 'Inbound rule for WinRM'
         }
       end
 
       it do
         expect do
           is_expected.to contain_exec('set rule Windows Remote Management')
-        end.not_to raise_error
+        end.to raise_error(Puppet::Error)
```

Comment:
this should include the error

```ruby
raise_error(Puppet::Error, /regex for error message/)
```
This can be found by putting in `/gibberish/`, running the test, then seeing what the error is.